generators/builtins/unsignedTypes.kt

```diff
@@ -579,7 +584,12 @@ internal constructor(
 
     override fun iterator(): ${elementType}Iterator = ${elementType}ProgressionIterator(first, last, step)
 
-    /** Checks if the progression is empty. */
+    /** 
+     * Checks if the progression is empty.
+     
+     * Progression with positive step is empty if its first element is greater than the last element.
```

Comment:
Either here should be an article as in Progression docs, "... with _a_ positive step". Or vice versa.

Reply:
Oops. Fixed 🙂